### DISCUSSION TITLE
Bandwidth to work on the site

### DISCUSSION THREAD

**Discussion by nmelhado** [nmelhado/league-page#300] [Announcements]
Hi all!

I've had very little bandwidth in the last 6 months to work on this, but I have a rare week coming up with a little bit of time.

If anyone has any specific requests or ideas, drop them here and I'll try to get some improvements added in the next week!

**Comment by Drewmaloney** [comment]
Hi, 

Thanks for making updates, I really appreciate this. I debuted the site to our league before our draft last week and they're obsessed with it. 
I was thinking about adding some stuff, I'm still learning so things obviously take a long time for me. So, I'm not sure if things on this list are super involved or relatively easy for someone like yourself who has a lot of experience. 

Some of the suggestions/things I was looking to add were:

-Highlighting matchup of the week on the matchups page. Maybe make it appear at the top of the list with a little graphic or something. Could choose it just by the 2 teams who have the highest projected score or something like that. Haven't really fleshed it out too much on the details but something I was thinking about. Feels like this wouldn't be too much of a lift. (Perhaps an additional feature on the homepage as well). I  Understand that the matchup could change throughout the week based on starters/injuries/etc... which would be fine. Also thought of maybe a rival ranking in the background, where you can assign a value to every person in the league to every other person and key the matchup off that just by adding up rival scores and featuring the one with the lowest. So, for me, I'd have one guy who is my biggest rival be 1, my least rival 11. Would prob want that to be customizable so you can take into account more than just strictly fantasy points. IDK, this is becoming a stream of consciousness now. I'll stop. 

-A prediction system or something like that. Before season starts people can pick things like, winner, most points, least points, most trades, most waiver wire moves. We're big on side pots in our league and the commish has normally just done this on his own and kept track of an excel sheet. Which, I'll probably just end up embedding on the site somewhere. 

-A little bit of a piggy back and also stealing the ideal from Yahoo, but a weekly pick 'em that tracks everyone who participates records. Was thinking of using sql in the background. I created something a little bit similar in python on years back so I was going to try to revisit that (I only have previous experience with Python [unless you count MATLAB too which I don't think actually counts]) but wasn't sure if that would be the best route. I'm still very new to JS and I'm sure it can handle something like this with ease but I'd be starting from 0 basically so not sure how involved that would be. 

Just some thoughts, let me know what you think. If these would be terrible ideas, too difficult, extremely easy, whatever. I'm fine with being told they're dumb so no need to hold back. haha. 

Thanks,
Drew
